app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -73,116 +79,85 @@ keywords:
 
-* have [indefinite leave to remain](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk) 
+* the job offers visa sponsorship 
 
-You will not need to be sponsored for a Skilled Worker visa if you have a: 
+* the salary offered meets the requirement for a Skilled Worker visa 
 
-* [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa) 
+$teachvacancies$
 
-* [Family visa](https://www.gov.uk/uk-family-visa) 
+## Salary requirements for a Skilled Worker visa
 
-* [Graduate visa](https://www.gov.uk/graduate-visa) 
+When you apply for a job, you must make sure the salary offered meets the requirement for a Skilled Worker visa.  
 
-* [High Potential Individual visa (HPI)](https://www.gov.uk/high-potential-individual-visa)
+The salary you need for a Skilled Worker visa will depend on whether you are employed as a qualified or unqualified teacher.  
 
-* [India Young Professionals Scheme visa](https://www.gov.uk/india-young-professionals-scheme-visa) 
+To work as a qualified teacher in England, you need English qualified teacher status (QTS). If you do not have QTS, you will not be qualified to teach in some schools in England. The details of the job you are applying for will include whether the role is for teachers with QTS.  
 
-* [UK Ancestry visa](https://www.gov.uk/ancestry-visa) 
+The salary requirement varies depending on the area of England you are applying to work in.  
 
-* [Youth Mobility Scheme visa](https://www.gov.uk/youth-mobility) 
+Schools have some flexibility in how they set salaries. You may be able to negotiate a different rate of pay if the salary offered does not meet the Skilled Worker visa requirements. 
 
-## Visa fees and other charges
+$salaryrequirements$
 
-You will usually need to show proof of a certain level of savings or income. You’ll be guided through this process when you apply for your visa.
+### Working part time as a teacher
 
-### Visa fees
+You can work part-time, but your salary must meet the threshold for a Skilled Worker visa. 
 
-You’ll also have to [pay visa fees](https://www.gov.uk/visa-fees).
+There are 2 requirements:
 
-### Immigration Health Surcharge 
+* your part-time salary must be at least $nonuk_skilledworkervisateacherminparttime$
+* the full-time equivalent of your salary must meet the minimum set for your region and status for qualified and unqualified teachers
 
-You may need to [pay for UK healthcare](https://www.gov.uk/healthcare-immigration-application) as part of your immigration application – this is called the Immigration Health Surcharge (IHS).
+Schools have some flexibility in how they set salaries. You may be able to negotiate a different rate of pay if the salary on offer does not meet the Skilled Worker visa requirements.
 
-$teachinuk$
+## Dependants
 
-## Contact
+You may be able to [bring your partner and children with you on a Skilled Worker visa](https://www.gov.uk/skilled-worker-visa/your-partner-and-children).
```

Comment:
Is this link text slightly misleading - just reading the link text, it implies I will be able to - it is the bit outside the link text that says `may` ?